go/obsclient/authclient.go

```diff
@@ -30,6 +31,7 @@ func NewAuthObsClient(client *rpc.EncRPCClient) *AuthObsClient {
 		ObsClient: ObsClient{
 			RPCClient: client,
```

Comment:
@BedrockSquirrel I think this field can be private too (if you use the `NewObsClient` constructor in the tests)?